features/suppliers.feature

```diff
@@ -38,20 +38,23 @@ Scenario: Create a Supplier
     Then I should see "123 Test Road" in the "Address" field
     Then I should see "8005551234" in the "phone_number" field
 
-# Scenario: List all pets
-#     When I visit the "Home Page"
-#     And I press the "Search" button
-#     Then I should see "fido" in the results
-#     And I should see "kitty" in the results
-#     And I should not see "leo" in the results
+Scenario: List all suppliers
+    When I visit the "Home Page"
+    And I press the "List" button
+    Then I should see "Catherine" in the results
+    And I should see "Evan" in the results
+    And I should see "Bea" in the results
+    And I should see "Sam" in the results
 
-# Scenario: List all dogs
-#     When I visit the "Home Page"
-#     And I set the "Category" to "dog"
-#     And I press the "Search" button
-#     Then I should see "fido" in the results
-#     And I should not see "kitty" in the results
-#     And I should not see "leo" in the results
+Scenario: Search a Supplier
+    When I visit the "Home Page"
+    And I press the "Clear" button
+    And I set the "name" to "Evan"
+    And I press the "Search" button
+    Then I should see "Evan" in the "name" field
+    Then I should see "evan@titans.com" in the "email" field
+    Then I should see "14 Cashville Ln" in the "address" field
+    Then I should see "9991235575" in the "phone_number" field
```

Comment:
@sa3613 @evpoe should this also have a "Then I should NOT see..."?

Reply:
@CatherineRH good question but I'm having trouble figuring out what we would NOT want to see in this scenario...what're you thinking?

Reply:
We should not see Sophie, Catherine, Sam, or Bea right?

Reply:
@CatherineRH this is a good point, is it necessary to add? If yes I can add them. Let me know!

Reply:
it's a valid point but if we want to go that route I would recommend a different set of code...something like

`  Then I should see "Evan" in the results
    And I should not see "Sophie" in the results
    And I should not see "Catherine" in the results
    And I should not see "Sam" in the results
    And I should not see "Bea" in the results`

Reply:
the current test looks to see if the "fields" were populated correctly (a little redundant with our CREATE bdd test....while this updated version does a better job ensuring the "results" were populated correctly

two different tests but the version catherine is talking about might be more what we're going after!

Reply:
I'm fine with either also, but yes, Evan that's exactly what I was thinking

Reply:
On it